### PR TITLE
getMembers test

### PR DESCRIPTION
**Pull request by lumap**
i hope i didnt break anything

### CODE REVIEW

File: tests/constants.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const CACHED_COMMUNITY_GUILD_ID = 907350958810480671n;
+export const CACHED_COMMUNITY_GUILD_ID = 907350958810480671n; // revert back to 907350958810480671n when testing is finished
```
Comment (itohatweb) [inline tests/constants.ts:1]: remove this constant. Also maybe open an issue to move this to an env.

File: tests/mod.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ if (bot.cache.guilds.size() <= 10) {
 await delayUntil(10000, () => Boolean(startedAt));
 
 // CREATE ONE GUILD SO WE CAN REUSE LATER TO SAVE RATE LIMITS
-export const guild = await bot.helpers.createGuild({ name: "Discordeno Test" });
```
Comment (itohatweb) [inline tests/mod.ts:89]: revert this change

**Review by itohatweb — CHANGES_REQUESTED** [review]

File: tests/constants.ts (hunk 1 above)
Comment (lumap) [reply tests/constants.ts:1]: that const was already here and i bet its used in other places too

File: tests/mod.ts (hunk 2 above)
Comment (lumap) [reply tests/mod.ts:89]: woops

Comment (lumap) [reply tests/mod.ts]: done

File: tests/constants.ts (hunk 1 above)
Comment (itohatweb) [reply tests/constants.ts:1]: Still remove the comment, I might change to env then

**Review by Skillz4Killz — APPROVED** [review]

**Comment by Skillz4Killz** [comment]
run-test

**Comment by Skillz4Killz** [comment]
run-tests
